tox_packet/src/relay/onion_request.rs

```diff
@@ -15,11 +16,11 @@ use nom::combinator::rest;
 
 /// Encrypted payload should contain `IpPort`, `PublicKey` and inner encrypted
 /// payload that should contain at least `IpPort` struct.
-const ONION_MIN_PAYLOAD_SIZE: usize = (SIZE_IPPORT + MACBYTES) * 2 + PUBLICKEYBYTES;
+const ONION_MIN_PAYLOAD_SIZE: usize = (SIZE_IPPORT + <SalsaBox as AeadCore>::TagSize::USIZE) * 2 + crypto_box::KEY_SIZE;
 
 /// `OnionRequest1` packet with encrypted payload from `OnionRequest` packet
 /// shouldn't be bigger than `ONION_MAX_PACKET_SIZE`.
-const ONION_MAX_PAYLOAD_SIZE: usize = ONION_MAX_PACKET_SIZE - (1 + NONCEBYTES + PUBLICKEYBYTES + ONION_RETURN_1_SIZE);
```

Comment:
Do we really have to have NONCEBYTES but not PUBLICKEYBYTES? Why not create an alias to crypto_box::KEY_SIZE?

Reply:
To be honest I thought we had tox_crypto crate to be able to keep all these consts in one place.

Reply:
> To be honest I thought we had tox_crypto crate to be able to keep all these consts in one place.

It didn't work this way because in tox we have both symmetric and asymmetric crypto, and different kind of hashes. So there are a lot of consts that have same or similar name, but we had here only consts for asymmetric crypto. I'd actually prefer not to redefine consts and import them directly from tox-crypto lib because this way we won't have a confusion - before we used both sodiumoxide and our redefined consts at the same time in different places. As for NONCEBYTES ryst-crypto just don't have a convenient const to use.